recipes/default.rb

```diff
@@ -23,6 +23,7 @@
 poise_service_user user do
   group group
```

Comment:
Thanks @chen01.

Could you also add the `extend` line here as well? Since we're using a method from the helpers.